### PR TITLE
Add w3m test case and fips tests entry

### PR DESCRIPTION
**Pull request by jouyingbin**
Add w3m.pm test case was located in console/w3m.pm
Add w3m.pm entry in load_fips_tests_web() in main.pm

### CODE REVIEW

File: tests/console/w3m.pm [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    send_key "q";
+    sleep 1;
+    send_key "y";
+
+    send_key "ctrl-l";
+    script_run "w3m https://build.opensuse.org", 0;
+    assert_screen "w3m-connect-OBS-webpage";
+    send_key "q";
+    sleep 1;
+    send_key "y";
```
Comment (okurz) [inline tests/console/w3m.pm]: [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html).

File: tests/console/w3m.pm [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+# without any warranty.
+
+# Case 1525204 - FIPS: w3m
+
+use base "consoletest";
+use strict;
+use testapi;
+
+sub run() {
+    my $name = ref(@_[0]);
```
Comment (okurz) [inline tests/console/w3m.pm]: where did you copy-paste this from? I don't see you using any variable and this construct looks weird.

Comment (jouyingbin) [inline tests/console/w3m.pm]: @okurz , hi Oliver, thanks for highlight, I just want to eliminate it.

File: tests/console/w3m.pm (hunk 1 above)
Comment (jouyingbin) [inline tests/console/w3m.pm]: @okurz , thanks Oliver , I will rewrite the code again using extract method :)

**Comment by jouyingbin** [comment]
@okurz , I have modified the case again, would you please help to review it again? Thanks a lot.

local test : http://147.2.211.160/tests/610/modules/w3m/steps/1
I marked the line[1]  while testing today because of the website is not accessible now from Taipei office.
BTW, there are 3 needles need to be submitted to gitlab later.

[1] suse => "https://www.suse.com/"

File: tests/console/w3m.pm [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+use strict;
+use testapi;
+
+sub run() {
+    select_console "root-console";
+
+    assert_script_run('rpm -q w3m');
+    script_run("zypper --no-refresh se -it pattern fips");
+
+    #https_url
```
Comment (okurz) [inline tests/console/w3m.pm]: delete

File: tests/console/w3m.pm [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    my %https_url = (
+        google => "https://www.google.com/",
+        suse   => "https://www.suse.com/",
+        OBS    => "https://build.opensuse.org/",
+    );
+
+    for my $p (keys %https_url) {
+        send_key "ctrl-l";
+        script_run "w3m $https_url{$p}", 0;
+        assert_screen("w3m-connect-$p-webpage", 30);
```
Comment (okurz) [inline tests/console/w3m.pm]: that's the default timeout, just use
`assert_screen "w3m-connect-$p-webpage;`

**Comment by okurz** [comment]
just two minor points. Tell me if I get annoying and I change and merge myself :-)

File: tests/console/w3m.pm (hunk 3 above)
Comment (jouyingbin) [inline tests/console/w3m.pm]: Done! :)

File: tests/console/w3m.pm (hunk 4 above)
Comment (jouyingbin) [inline tests/console/w3m.pm]: Okay, I just amend it again, thanks. :)

**Comment by jouyingbin** [comment]
@okurz , not annoying at all :) I also want to amend it as more accurate. Would you mind checking it up again?  Thanks a lot for your suggestion.

**Comment by jouyingbin** [comment]
I have modified the commit to fix the conflict, would it be possible to be reviewed again ? Thanks a lot. :-)

File: tests/console/w3m.pm [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    script_run("zypper --no-refresh se -it pattern fips");
+
+    my %https_url = (
+        google => "https://www.google.com/",
+        suse   => "https://www.suse.com/",
+        OBS    => "https://build.opensuse.org/",
+    );
+
+    for my $p (keys %https_url) {
+        send_key "ctrl-l";
```
Comment (okurz) [inline tests/console/w3m.pm]: Please use `type_string 'clear\n';` here instead of `ctrl-l`, see 8168ab04899ef17f8cc48df760c955635ffec8f5 for reasoning.

**Comment by okurz** [comment]
one remark, did not see anything else.

**Comment by jouyingbin** [comment]
I apologize that I mix up the two commit in one PR. Sorry about it , I will resend it again.

**Pull request closed without merging**